docs/source/index.rst

```diff
@@ -31,6 +31,13 @@ DataFusion offers SQL and Dataframe APIs, excellent
 CSV, Parquet, JSON, and Avro, extensive customization, and a great
 community.
 
+The `example usage`_ section in the user guide and the `datafusion-examples`_ code in the crate contain information on using DataFusion.
```

Comment:
This makes the readme from github consistet with the readme on https://arrow.apache.org/datafusion/

<img width="943" alt="Screenshot 2023-05-28 at 7 34 36 AM" src="https://github.com/apache/arrow-datafusion/assets/490673/e8637b22-137d-4a49-88cb-0e75a7f178a1">